tox.ini

```diff
@@ -64,57 +67,65 @@ deps =
     mock
     nose
     hypothesis>=3.5.2
+    {env:TOX_EXTRA_DEPS:}
 commands =
-    pytest -n1 -ra {posargs:testing}
+    {env:TOX_COVERAGE_RUN:} pytest -n1 -ra {posargs:testing}
 
 [testenv:py36-xdist]
 deps = {[testenv:py27-xdist]deps}
 commands =
-    pytest -n3 -ra {posargs:testing}
+    {env:TOX_COVERAGE_RUN:} pytest -n3 -ra {posargs:testing}
 
 [testenv:py27-pexpect]
 changedir = testing
 platform = linux|darwin
-deps = pexpect
+deps =
+    pexpect
+    {env:TOX_EXTRA_DEPS:}
 commands =
-    pytest -ra test_pdb.py test_terminal.py test_unittest.py
+    {env:TOX_COVERAGE_RUN:} pytest -ra test_pdb.py test_terminal.py test_unittest.py
```

Comment:
Looking at the Travis build, `py27/py36-trial` seem to be missing this configuration